client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table/index.tsx

```diff
@@ -12,22 +16,69 @@ interface Props {
 }
 
 export default function SiteTable( { isLoading, columns, items }: Props ) {
+	//const initialText = 'Edit All';
+	const translate = useTranslate();
+	const [ isBulkManagementActive, setIsBulkManagementActive ] = useState( true );
+
+	/**
+	 * this logic take the existing state and inverts it,
+	 * I've used the letter b because it's a Boolean
+	 */
+	const toggleBulkEditMode = () => setIsBulkManagementActive( ( b ) => ! b );
+
+	// Actions
+
 	return (
 		<table className="site-table__table">
 			<thead>
-				<tr>
-					{ columns.map( ( column, index ) => (
-						<th key={ column.key }>
-							{ index === 0 && (
-								<Icon className="site-table__favorite-icon" size={ 24 } icon={ starFilled } />
-							) }
-							<span className={ classNames( index === 0 && 'site-table-site-title' ) }>
-								{ column.title }
-							</span>
+				{ isBulkManagementActive ? (
+					<tr>
+						{ columns.map( ( column, index ) => (
+							<th key={ column.key }>
+								{ index === 0 && (
+									<Icon className="site-table__favorite-icon" size={ 24 } icon={ starFilled } />
+								) }
+
+								<span className={ classNames( index === 0 && 'site-table-site-title' ) }>
+									{ column.title }
+								</span>
+							</th>
+						) ) }
+						<th>
+							<div className="plugin-common-table__bulk-actions">
+								<ButtonGroup className="edit-all__bulk-actions">
+									<Button compact onClick={ toggleBulkEditMode }>
+										{ translate( 'Edit All', { context: 'button label' } ) }
+									</Button>
+								</ButtonGroup>
+							</div>
 						</th>
-					) ) }
-					<th></th>
-				</tr>
+					</tr>
+				) : (
+					<tr>
+						<ButtonGroup className="edit-all__bulk-actions-pause-monitor">
```

Comment:
We can wrap everything inside the `tr` in a `th` tag with `colSpan={ 100 }` and `text-align: right`

```
<tr>
    <th colSpan={100}>
       <ButtonGroup>
       ..........
   </th>
</tr>
```

Reply:
We could take some inspiration from https://github.com/Automattic/wp-calypso/blob/add/jetpack-dashboard-downtime-monitoring-bulk-action-menu/client/my-sites/plugins/plugin-list-header/index.jsx. This is how it is being done in the Plugin's page where we switch to a dropdown in smaller screens.

Reply:
Exactly what I used as a model. Thanks!